mr/build.gradle

```diff
@@ -52,8 +52,8 @@ project(':iceberg-mr') {
       exclude group: 'org.pentaho' // missing dependency
       exclude group: 'org.slf4j', module: 'slf4j-log4j12'
     }
-    compileOnly libs.hive2.metastore
-    compileOnly libs.hive2.serde
+    compileOnly libs.hive3.metastore
```

Comment:
Sir. should we jump straight to HIVE4?

Reply:
Please share your thoughts in the discussion thread on the dev list https://lists.apache.org/thread/zg14b8cor4lnbyd3t4n1297y2bwb1fsg.